dashboard/app/controllers/levels_controller.rb

```diff
@@ -43,7 +43,7 @@ def edit_blocks
     @level = Level.find(params[:level_id])
     authorize! :edit, @level
     type = params[:type]
-    blocks_xml = @level.properties[type].presence || @level[type]
+    blocks_xml = @level.properties[type].presence || @level[type] || '<xml></xml>'
```

Comment:
'<xml></xml>' feels like we're more explicitly saying we want no xml blocks. Empty string seems easier to confuse the intentions and think maybe we just don't want this value set yet.

I think <xml></xml> would probably be cleaner in StudioApp as well.